### PR TITLE
lint errors fixed

### PR DESCRIPTION
**Pull request by ElenaBorisenkoO**

### CODE REVIEW

File: src/tests/filter.test.js [hunk 1]
```diff
@@ -89,6 +88,7 @@ describe('tests for method filter', () => {
     const checkNumericRange = function(value) {
       return typeof value === 'number' && (value >= this.minimum && value <= this.maximum);
```
Comment (Jarosic) [inline src/tests/filter.test.js:89]: may be would be beter use `if() else {} ? `

**Review by Jarosic — APPROVED** [review]
good job!

File: src/tests/filter.test.js [hunk 2]
```diff
@@ -11,12 +11,11 @@ describe('tests for method filter', () => {
   test('instance has not Own Property filter', () => {
     const arr = new MyArray(1, 4, 0);
 
-    expect(arr.hasOwnProperty('filter')).toBeFalsy();
+    expect(arr.Object.prototype.hasOwnProperty.call('filter')).toBeFalsy();
```
Comment (AndrZakhara) [inline src/tests/filter.test.js]: invalid sintax, you need Object.prototype.hasOwnProperty.call(context, prop)

File: src/tests/filter.test.js (hunk 1 above)
Comment (ElenaBorisenkoO) [reply src/tests/filter.test.js:89]: My job was just fixing lint errors. It's not my method:)

File: src/tests/find.test.js [hunk 3]
```diff
@@ -44,7 +42,7 @@ describe('tests for method find', () => {
   test('instance has not Own Property map', () => {
     const arr = new MyArray(1, 4, 0);
 
-    expect(arr.hasOwnProperty('find')).toBeFalsy();
+    expect(arr.Object.prototype.hasOwnProperty.call('find')).toBeFalsy();
```
Comment (AndrZakhara) [inline src/tests/find.test.js]: invalid sintax, you need Object.prototype.hasOwnProperty.call(context, prop)

File: src/tests/forEach.test.js [hunk 4]
```diff
@@ -14,7 +14,7 @@ describe('tests for method forEach', () => {
   });
 
   test('instance has not Own Property forEach ', () => {
-    expect(arr.hasOwnProperty('forEach')).toBeFalsy();
+    expect(arr.Object.prototype.hasOwnProperty.call('forEach')).toBeFalsy();
```
Comment (AndrZakhara) [inline src/tests/forEach.test.js]: invalid sintax, you need Object.prototype.hasOwnProperty.call(context, prop)

File: src/tests/from.test.js [hunk 5]
```diff
@@ -1,22 +1,22 @@
...
   });
 
-  test("Instance has not Own Property FROM", () => {
+  test('Instance has not Own Property FROM', () => {
     const arr = new MyArray(1, 4, 0);
 
-    expect(MyArray.hasOwnProperty("from")).toBeTruthy();
-    expect(arr.hasOwnProperty("from")).toBeFalsy();
+    expect(MyArray.Object.prototype.hasOwnProperty.call('from')).toBeTruthy();
+    expect(arr.Object.prototype.hasOwnProperty.call('from')).toBeFalsy();
```
Comment (AndrZakhara) [inline src/tests/from.test.js]: invalid sintax, you need Object.prototype.hasOwnProperty.call(context, prop)

File: src/tests/map.test.js [hunk 6]
```diff
@@ -11,7 +11,7 @@ describe('tests for method map', () => {
   test('instance has not Own Property map', () => {
     const arr = new MyArray(1, 4, 0);
 
-    expect(arr.hasOwnProperty('map')).toBeFalsy();
+    expect(arr.Object.prototype.hasOwnProperty.call('map')).toBeFalsy();
```
Comment (AndrZakhara) [inline src/tests/map.test.js]: invalid sintax, you need Object.prototype.hasOwnProperty.call(context, prop)

File: src/tests/filter.test.js (hunk 1 above)
Comment (Jarosic) [reply src/tests/filter.test.js:89]: Ok)

File: src/tests/pop.test.js [hunk 7]
```diff
@@ -14,7 +14,7 @@ describe('tests for method pop', () => {
   });
 
   test('instance has not Own Property pop', () => {
-    expect(arr.hasOwnProperty('pop')).toBeFalsy();
+    expect(arr.Object.prototype.hasOwnProperty.call('pop')).toBeFalsy();
```
Comment (AndrZakhara) [inline src/tests/pop.test.js]: invalid sintax, you need Object.prototype.hasOwnProperty.call(context, prop)

File: src/tests/push.test.js [hunk 8]
```diff
@@ -14,7 +14,7 @@ describe('tests for method push', () => {
   });
 
   test('instance has not own property push', () => {
-    expect(arr.hasOwnProperty('push')).toBeFalsy();
+    expect(arr.Object.prototype.hasOwnProperty.call('push')).toBeFalsy();
```
Comment (AndrZakhara) [inline src/tests/push.test.js]: invalid sintax, you need Object.prototype.hasOwnProperty.call(context, prop)

File: src/tests/reduce.test.js [hunk 9]
```diff
@@ -1,39 +1,41 @@
...
 
     expect(arr.reduce).toBeInstanceOf(Function);
   });
 
-  test("instance has not own property reduce", () => {
+  test('instance has not own property reduce', () => {
     const arr = new MyArray(1, 4, 0);
 
-    expect(arr.hasOwnProperty("reduce")).toBeFalsy();
+    expect(arr.Object.prototype.hasOwnProperty.call('reduce')).toBeFalsy();
```
Comment (AndrZakhara) [inline src/tests/reduce.test.js]: invalid sintax, you need Object.prototype.hasOwnProperty.call(context, prop)

File: src/tests/spread.test.js [hunk 10]
```diff
@@ -7,15 +7,14 @@ describe('tests for spread', () => {
 
     expect(arr[Symbol.iterator]).toBeInstanceOf(Function);
   });
-  
   test('instance has not own property [Symbol.iterator]', () => {
     const arr = new MyArray(1, 2, 3);
 
-    expect(arr.hasOwnProperty('Symbol.iterator')).toBeFalsy();
+    expect(arr.Object.prototype.hasOwnProperty.call('Symbol.iterator')).toBeFalsy();
```
Comment (AndrZakhara) [inline src/tests/spread.test.js]: invalid sintax, you need Object.prototype.hasOwnProperty.call(context, prop)

**Review by AndrZakhara — CHANGES_REQUESTED** [review]
Fix bugs, and send your code again.

**Comment by AndrZakhara** [comment]
approved

**Comment by AndrZakhara** [comment]
approved

**Review by AndrZakhara — APPROVED** [review]

**Pull request merged**